examples/asr/conf/ssl/citrinet/citrinet_ssl_1024.yaml

```diff
@@ -402,22 +402,22 @@ model:
         se_context_size: ${model.model_defaults.se_context_size}
         kernel_size_factor: ${model.model_defaults.kernel_size_factor}
 
-
   decoder:
     _target_: nemo.collections.asr.modules.ConvASRDecoderReconstruction
     feat_in: ${model.model_defaults.enc_final}
     feat_hidden: 128
     feat_out: ${model.model_defaults.decoder_out_channels}
     stride_layers: 1
-
+    non_stride_layers: 0
```

Comment:
It would be nice to have comments across the values of this config to understand what are legal values and what they affect